src/compose.js

```diff
@@ -30,4 +30,6 @@ var _createComposer = require('./internal/_createComposer');
  *      //≅ triple(double(square(5)))
  *      squareThenDoubleThenTriple(5); //=> 150
  */
-module.exports = _createComposer(_compose);
+module.exports = function compose() {
+  return pipe.apply(this, reverse(arguments));
```

Comment:
Decades of exposure to written English (with no countervailing exposure to a right-to-left language) has had an effect on my neural pathways: I now find it more natural to follow things that read left to right rather than right to left.

Natural (for me):

              +-----+
    --- a --->|     |
              |  f  |                +-----+
    --- b --->|     |--- f(a, b) --->|     |
              +-----+                |  g  |                      +-----+
    --- c -------------------------->|     |--- g(f(a, b), c) --->|     |
                                     +-----+                      |  h  |
    --- d ------------------------------------------------------->|     |--- h(g(f(a, b), c), d) --->
                                                                  +-----+

Unnatural (for me):

                                                                                    +-----+
                                                                                    |     |<--- a ---
                                                             +-----+                |  f  |
                                                             |     |<--- f(a, b) ---|     |<--- b ---
                                +-----+                      |  g  |                +-----+
                                |     |<--- g(f(a, b), c) ---|     |<-------------------------- c ---
                                |  h  |                      +-----+
    <--- h(g(f(a, b), c), d) ---|     |<------------------------------------------------------- d ---
                                +-----+